scripts/run_local.sh

```diff
@@ -54,4 +54,4 @@ FIGHT_HEALTH_ADMIN_USER="admin" FIGHT_HEALTH_ADMIN_PASSWORD="admin" python manag
 
 python manage.py make_user  --username test_user --domain farts --password farts2
 
-RECAPTCHA_TESTING=true OAUTHLIB_RELAX_TOKEN_SCOPE=1 uvicorn fighthealthinsurance.asgi:application --reload --reload-dir fighthealthinsurance --reload-exclude "*.pyc,__pycache__/*,*.pyo,*~,#*#,.#*" --access-log --log-config conf/uvlog_config.yaml --port 8000 --ssl-keyfile key.pem --ssl-certfile cert.pem $@
+./manage.py runserver
```

Comment:
We probably still need to set RECAPTCHA_TESTING=true OAUTHLIB_RELAX_TOKEN_SCOPE=1 & we probably need to configure SSL magics ala daphne -e ssl:443:privateKey=key.pem:certKey=crt.pem django_project.asgi:application